lib/service/businessLogic.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const kinveyCompletionHandler = require('./kinveyCompletionHandler');
+const kinveyErrors = require('kinvey-datalink-errors');
+const notImplementedHandler = require('./notImplementedHandler');
+
+const registeredFunctions = new Map();
+
+function register(taskName, functionToExecute) {
```

Comment:
Datalinks implement this as a class; BL does it as a series of exported methods.  Any particular reason for the difference?